spring-test/src/test/java/org/springframework/test/context/jdbc/RepeatableSqlAnnotationSqlScriptsTests.java

```diff
@@ -37,6 +37,7 @@
  */
 @FixMethodOrder(MethodSorters.NAME_ASCENDING)
 @ContextConfiguration(classes = EmptyDatabaseConfig.class)
+@Sql("drop-schema.sql")
```

Comment:
We will need to introduce a new, separate test class to test the new feature. (see previous comments)